aws/main.tf

```diff
@@ -0,0 +1,60 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.0"
+    }
+  }
+}
+
+locals {
+  yaml_secrets= yamldecode(file("../secrets.yaml"))
+}
+
+# Configure the AWS Provider
+provider "aws" {
+  region = "us-east-2"
+}
+
+resource "aws_ssm_parameter" "github_pat" {
+  name  = "/github/pat"
+  type  = "String"
+  value = local.yaml_secrets.samples.github_pats[0]
+}
+
+resource "aws_ssm_parameter" "aws_access_key_id" {
```

Comment:
Should we use a single parameter for both AWS Access Key ID and Secret Key?

Reply:
You think a string list for that then?

Reply:
It could just be a string with `\n` as separator. Can you please check `/dev/aws/credentials` parameter in us-east-1?

Reply:
okay. yeah. let me check. I'll mimic whatever is there.